src/index.ts

```diff
@@ -48,6 +69,21 @@ const getLogin = async (octokit: Octokit): Promise<string> => {
   return login;
 }
 
+// If repoConfig (the file repositories.json) exists, read it and return the list of repositories from the repositories key
+let repoConfig = 'repositories.json'
+const getRepositoriesFromJson = async (): Promise<Repository[]> => {
+  const fs = await import('fs').then(m => m.default);
+  const path = await import('path').then(m => m.default);
+  const repositoriesPath = path.join(process.cwd(), repoConfig);
+  if (fs.existsSync(repositoriesPath)) {
+    const repositories = JSON.parse(fs.readFileSync(repositoriesPath, 'utf8')).repositories;
```

Comment:
We should check the file is well formatted (both in terms of being valid JSON, and actually being an array of `Repository` objects), and throwing a useful error if not.